apps/researcher/src/app/[locale]/communities/[slug]/actions.ts

```diff
@@ -1,6 +1,23 @@
 'use server';
 
-import {joinCommunity} from '@/lib/community';
+import {joinCommunity, updateDescription} from '@/lib/community';
+import {revalidatePath} from 'next/cache';
+
+interface UpdateDescriptionActionProps {
+  communityId: string;
+  communitySlug: string;
+  description: string;
+}
+
+export async function updateDescriptionAction({
```

Comment:
If I understand this correctly, there are two server actions in this file, on line 12 and on line 23. Their names differ, though: the first one uses the `Action` suffix. Is there a way to harmonize these names?

Reply:
Yes, this is correct. I will use the suffix `Action` for both actions and differentiate it from the functions in '@/lib/community'.

Reply:
See: https://github.com/colonial-heritage/colonial-collections/pull/274/commits/e4243df16d1b52fc81e64d966616d2a70d8dd81a

Reply:
Thanks! Some considerations:

1. The `Action` suffix is somewhat redundant because the functions are in a file called `actions.ts` that make the scope of the functions clear?
2. You could argue that all functions are 'actions', so adding the suffix `Action` doesn't say that much?
3. Consistency: if we add the suffix `Actions` to server actions, should we also add the suffix `Component` to components?

Does Next.js have a convention for naming server actions that we can follow?

Reply:
Next.js does not have a convention for naming server actions. In their docs, I also see some actions with the suffix 'Action' and some without.

I added the suffix so it is clear that it was different from the functions in '@/lib/community' with similar names. But then again the file name (`actions.ts`) also makes that clear. I don't really have an opinion about this subject. I will remove the suffix 'Action'.

Reply:
I looked at this file some more. `joinCommunity` was one of the first server actions. But this function should also revalidate pages, without this you won't see your avatar in the member list, and the member count can be wrong. Also, it has `organizationId` as input in the rest of the community logic I use the term `communityId`. So I made some changes to this action.

Reply:
The code is updated. I named the actions `updateDescriptionAndRevalidate` and `joinCommunityAndRevalidate`